### DISCUSSION TITLE
Item bug 💀

### DISCUSSION THREAD

**Discussion by thegreatestbrian** [Melledy/Grasscutter#1389] [Q&A] [answered]
Guys i type /give 101205 90 to get 90 Naku Weed, but how it just give me only 1?

**Comment by ghost** [comment, marked as answer]
Use `/give 101205 x90`

![image](https://user-images.githubusercontent.com/93566768/175905969-cb1c40ea-f4ac-4ab7-95bf-02f28748f94d.png)

**Reply by thegreatestbrian** [reply]
W

**Reply by Birdulon** [reply]
See https://github.com/Grasscutters/Grasscutter/wiki/Commands#give-command for more tips on the new give command!
